source/jobs/data-verification/use-data-verification.txt

```diff
@@ -10,30 +10,30 @@ Use Data Verification
    :depth: 1
    :class: singlecol
 
-Data verification runs as a separate process that is tightly
+Data verification optionally runs as a separate process that is tightly
 coupled to a sync job. Data verification can be enabled for one-time 
 sync jobs. This page details how to enable and debug the 
 data verification process.
 
 Enable Data Verification
 ------------------------
 
-You can enable data verification when :ref:`creating a sync job <rm-create-jobs>`
-on the :guilabel:`Data Migration` tab.
+You can optionally enable data verification when 
+:ref:`creating a sync job <rm-create-jobs>` on the :guilabel:`Data Migration` tab.
 
 On the :guilabel:`Migration Options` module, click the 
 :guilabel:`Verify migrated data` check box for your sync job.
 
-.. note:: Data Verification Duration
+.. note::
 
    The Data Verification process can take as long or longer than the data sync job.
    You can click :guilabel:`Terminate` to cancel a Data Verification process.
 
 To view status and metadata information about the data verification process
-for a sync job, click the :guilabel:`SYNC JOB / DATA VERIFICATION` slider.
+for a sync job, click the :guilabel:`Data Verification` dropdown:
```

Comment:
(terminology nit -- [drop-down](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#d)) 
Drop-down should be used as an adjective only, (so drop-down menu, rather than drop-down) and it should always be hyphenated. Would you mind doing a quick search/replace for this?